src/main/java/dp/api/dataset/models/Metadata.java

```diff
@@ -231,7 +258,16 @@ public void setQmi(GeneralDetails qmi) {
     }
 
     public String getReleaseDate() {
-        return releaseDate;
+
+        try {
```

Comment:
While this does work it would be "proper" to use a time library see: https://stackoverflow.com/questions/12042352/retrieve-month-day-and-year-values-from-a-string-using-java

Splitting strings and using array indexes introduces the risk of index out of bounds exceptions.
The `joda` time lib probably has a method for getting the month a string as well